### DISCUSSION TITLE
OpenRadioss Linear Static Implicit Problem Capability

### DISCUSSION THREAD

**Discussion by anoopngmys** [OpenRadioss/OpenRadioss#1705] [Q&A]
Hey Everyone,

I am new to OpenRadioss and I am trying to solve linear static implicite problem. I just wanted to know whether the solver has capability to solve "Linear Static Implicit Problem". 

Thanks.

**Comment by PaulAltair** [comment]
Hi, you need to specifically compile OpenRadioss with implicit features, only explicit are compiled by default, see [Discussion on compiling for Implicit](https://github.com/orgs/OpenRadioss/discussions/485) what is the interest to use the Implicit features?

**Reply by anoopngmys** [reply]
Hey Paul,

Thank you for the reply. I am using Visual Studio+one API Base Toolkit+one API HPC Toolkit to built openradioss in Windows.
Now, I want to build and link MUMPS, Lapac and Scalapac to engine.
Kindly let me know the steps to be followed to do this.

Thanks in advance.

**Reply by PaulAltair** [reply]
I've checked with developer colleagues, there is no way to build for implicit solving for the native Windows version in VisualStudio right now (the compiler scripts aren't set up to do it), if you need implicit on windows, the only way would be to use the wsl approach (not windows 'native', but effectively running the linux version in windows)
